src/secureCreatives.js

```diff
@@ -72,8 +72,8 @@ function resizeRemoteCreative({ adUnitCode, width, height }) {
   // resize both container div + iframe
   ['div', 'iframe'].forEach(elmType => {
     let elementStyle = getElementByAdUnit(elmType).style;
-    elementStyle.width = width;
-    elementStyle.height = height;
+    elementStyle.width = `${width}px`;
```

Comment:
Template Literals aren't supported on Internet Explorer:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals

Given the that browser support includes IE 10 and later:
https://github.com/prebid/Prebid.js/blob/master/README.md

This needs to be changed to not use a template literal

Reply:
Oh, I was assuming the build would transform it. Thank you for the info!

Reply:
@mike-chowla 
we do transform template literals. Thanks for checking!